velox/functions/sparksql/Register.cpp

```diff
@@ -178,6 +179,8 @@ void registerFunctions(const std::string& prefix) {
 
   registerFunction<JsonObjectKeysFunction, Array<Varchar>, Varchar>(
       {prefix + "json_object_keys"});
+  registerFunction<JsonArrayLengthFunction, int64_t, Varchar>(
```

Comment:
It seems the return type in Spark is int32, could you confirm?

https://github.com/apache/spark/blob/master/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/jsonExpressions.scala#L964

Reply:
Yes, it should be int32.